### DISCUSSION TITLE
Some plugin unwantedly maps L to `BufferLineCycleNext` - how to prevent?

### DISCUSSION THREAD

**Discussion by julien-h2** [LazyVim/LazyVim#4431] [Q&A] [answered]
Hello I just migrated to LazyVim and I added a line in my flash.nvim configuration to map `L` to `flash.treesitter`. 
The probem is some other plugin is overriding my key for normal mode and I can't find how to prevent it. 

The screenshot below shows that `L` is improperly mapped for normal mode. The file location is quite unhelpful. 

I have also searched the github repository and my local source code for `BufferLineCycleNext` and I have no matches. It's a mystery.

Does anyone know how to fix this?

![Screenshot 2024-09-13 023812](https://github.com/user-attachments/assets/55d2532a-9ba2-48b3-b769-4ef7163f7a58)

**Comment by nhld** [comment, marked as answer]
Hi, 
I did a repo search and this keymap comes from 'bufferline', [see](https://github.com/LazyVim/LazyVim/blob/3dbace941ee935c89c73fd774267043d12f57fe2/lua/lazyvim/plugins/ui.lua#L49).
Create `bufferline.lua` under `lua/plugins/` and add this:
```lua
return {
  "akinsho/bufferline.nvim",
  keys = {
    { "<S-l>", "" },
  },
}
```
In the docs ([see](https://www.lazyvim.org/configuration/plugins)), it says `{ "<S-l>", false },` to disable, I tried it but doesn't work. Hope this helps!

**Reply by julien-h2** [reply]
Thank you, I did not know that `{ "L", BufferLineCycleNext }` and `{ "<S-l>, BufferLineCycleNext  }` were equivalent.
